server/src/controllers/productController.js

```diff
@@ -1,56 +1,48 @@
-class productController {
-  products = [{
-    productId: 1,
-    productName: 'Item 1 ',
-    price: 19.99
-  },
-  {
-    productId: 2,
-    productName: 'Item2',
-    price: 25.99
-  }];
+import { v4 as uuidv4 } from 'uuid';
+class ProductController {
```

Comment:
I suggest to rename it to `ProductsController`, like endpoint is named: `/products`.
Also, `productRoutes` to `productsRoutes`.

Reply:
On my way :)